src/test-variations/test-variations.service.ts

```diff
@@ -80,20 +80,20 @@ export class TestVariationsService {
    * @returns
    */
   async find(
-    createTestRequestDto: BaselineDataDto & { projectId: string; sourceBranch?: string }
+    createTestRequestDto: BaselineDataDto & { projectId: string; baselineBranchName?: string }
```

Comment:
should work without renaming

Reply:
Yes, I renamed to prevent having two names for one thing, but it is not necessary. I could not find any usages of this field in the code so I decided that renaming should be safe. But we can recover original name to be 100% sure.

Reply:
it was used the same way, renaming is the only change here

Reply:
recovered the old name